modules/apps/asset/asset-categories-test/src/testIntegration/java/com/liferay/asset/categories/service/test/AssetCategoryLocalServiceTest.java

```diff
@@ -430,6 +429,54 @@ public void testCategoryWithLongNameIsTrimmed() throws Exception {
 		Assert.assertEquals(assetCategoryName, assetCategory.getName());
 	}
 
+	@Test
+	public void testCategoryWithLongTitlesAreTrimmed() throws Exception {
```

Comment:
Use AssetCategory in title pattern

Reply:
I'm following the pattern of testCategoryWithLongNameIsTrimmed.

in your opinion, what do you recommend?:
- change both to "assetCategory"
- change testCategoryWithLongNameIsTrimmed to testAddCategoryWithLongNameIsTrimmed (method name) and then split the new tests on testAddCategoryWithLongTitlesAreTrimmed and testUpdateCategoryWithLongTitlesAreTrimmed
- only change the new method.


Thanks for your input

Reply:
mmm, you are right.

Let's keep it as it is.